### ISSUE TITLE
[FE] Accessibility checks required for Staging review - Axe Scans in E2E testing

### ISSUE THREAD

**Issue by mshea0606** [department-of-veterans-affairs/va.gov-team#11275]
## Background
There are a few 508/accessibility checks required as a pre-requisite for the staging review. They are listed below.
[Axe scans in e2e testing](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#axe-scans-in-end-to-end-tests)

### Steps to test
1. Read the [basic end-to-end test setup](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/platform/quality-assurance/e2e-testing) document
2. Add [axe checks](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/testing/e2e/nightwatch-commands/axeCheck.js) to all of your end-to-end tests
3. Run tests on `localhost` to confirm proper functionality

### Expected result
Stable code with no violations reported in end-to-end tests.
Note: Axe checks that result in 1 or more violations will break branch builds and prevent merging into the `master` branch until they are resolved.

### Definition of done
No violations indicated in end-to-end tests.
OR
End-to-end tests result in violations, but those violations are documented in issue ticket(s). Issue tickets are listed as known issues in your staging review request ticket. 

## Tasks
- [ ] [Axe scans in end-to-end tests](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md#axe-scans-in-end-to-end-tests)

**Comment by mshea0606** [comment]
@erikphansen can you please add an estimate in here when you get a minute?

**Comment by erikphansen** [comment]
[This is the documentation for adding an aXe scan with Cypress](https://github.com/department-of-veterans-affairs/va.gov-team/blob/3958ace83553eac39f551f6b1b0b21956b63b95e/teams/vsp/teams/tools/frontend/cypress-migration-guide.md#cypress)
